.gitmodules

```diff
@@ -7,7 +7,7 @@
 	url = https://github.com/tommie/lx106-hal
 [submodule "esptool"]
 	path = esptool
-	url = https://github.com/pfalcon/esptool
+	url = https://github.com/themadinventor/esptool.git
```

Comment:
I don't plan to switch to upstream repo. See https://github.com/themadinventor/esptool/pull/89#issuecomment-203391199 why.